images/operator-alibabacloud/Dockerfile

```diff
@@ -0,0 +1,63 @@
+# syntax=docker/dockerfile:1.2
+
+# Copyright 2020-2021 Authors of Cilium
+# SPDX-License-Identifier: Apache-2.0
+
+ARG BASE_IMAGE=scratch
+ARG GOLANG_IMAGE=docker.io/library/golang:1.16.0@sha256:f3f90f4d30866c1bdae90012b506bd5e557ce27ccd2510ed30a011c44c1affc8
+ARG ALPINE_IMAGE=docker.io/library/alpine:3.13.1@sha256:08d6ca16c60fe7490c03d10dc339d9fd8ea67c6466dea8d558526b1330a85930
+
+# BUILDPLATFORM is an automatic platform ARG enabled by Docker BuildKit.
+# Represents the plataform where the build is happening, do not mix with
```

Comment:
> Please find some comments inline below, with one concern for pkg/ipam/crd.go, but mostly typos and nitpicks on comments otherwise.
> 
> Could we please have some documentation under Documentation/concepts/networking/ipam for the new IPAM mode? And ideally, some _Getting Started Guide_ to indicate how to install Cilium on AlibabaCloud?

thanks , i will fix this and add doc